src/Stripe.net/Entities/Identity/VerificationReports/VerificationReport.cs

```diff
@@ -0,0 +1,67 @@
+// File generated from our OpenAPI spec
+namespace Stripe.Identity
+{
+    using Newtonsoft.Json;
+
+    public class VerificationReport : StripeEntity<VerificationReport>, IHasId, IHasObject
+    {
+        /// <summary>
+        /// Unique identifier for the object.
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// String representing the object's type. Objects of the same type share the same value.
+        /// </summary>
+        [JsonProperty("object")]
+        public string Object { get; set; }
+
+        /// <summary>
+        /// Time at which the object was created. Measured in seconds since the Unix epoch.
+        /// </summary>
+        [JsonProperty("created")]
+        public long Created { get; set; }
```

Comment:
This is the wrong type right? Should it be a DateTime like https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Entities/Charges/Charge.cs#L185